frontend/src/api/login.ts

```diff
@@ -16,6 +16,7 @@ export interface RequestRegisterChain {
   name: string;
   description: string;
   address: string;
+  country_code: string;
```

Comment:
Country should be considered write only. 
So the country code should be able to be set from the api but not returned (except in the info route)

Reply:
fixed in 54e6f346223f0e170ae1a7ff22c07a198837f07f